example_test.go

```diff
@@ -106,17 +107,30 @@ func basicCreateAndPopulateKeyspace(t *testing.T, session gocqlx.Session) {
 
 	// Insert song using query builder.
 	insertSong := qb.Insert("examples.songs").
-		Columns("id", "title", "album", "artist", "tags", "data").Query(session)
```

Comment:
Please add new examples if appropriate.

Reply:
I remained your insert statement, just save it in the separate variable. I suppose it's a good example how one statement can be used in situations with all filled params and partially filled.

If you think it's inappropriate in this example, I can remove it.

Reply:
It break the convention, feel free to copy the example and fully adjust it to present your usecase.

Reply:
Ok, added separate example